packages/sku/src/context/defaultSkuConfig.ts

```diff
@@ -1,15 +1,15 @@
 import browserslistConfigSeek from 'browserslist-config-seek';
 import { join } from 'node:path';
 import isCompilePackage from '../lib/isCompilePackage.js';
+import type { SkuConfig } from '../../sku-types.d.ts';
 
-const defaultDecorator = (a) => a;
+const defaultDecorator = (a: any) => a;
```

Comment:
Should this have a comment explaining the any?

Reply:
This should be a T